### ISSUE TITLE
Add Custom SVG Icons to axis label in Horizontal bar chart

### ISSUE THREAD

**Issue by ARVINDTRONICS** [apexcharts/apexcharts.js#1582]
Hey,I was wondering if there is a way I could add custom SVG icons or font awesome icons to each of the axis labels in my horizontal bar chart for styling in apexcharts. Thanks for any help you can provide.

**Comment by kamilcglr** [comment]
Hello @ARVINDTRONICS , did you find a solution ?
Here is what the designer wants 
![Annotation 2020-06-20 011120](https://user-images.githubusercontent.com/36273997/85185191-470a1080-b293-11ea-8354-54aa8e33632d.png)

**Comment by ARVINDTRONICS** [comment]
@kamilcglr Nope,I had to convince the client to not have icons for this type of chart.

**Comment by kamilcglr** [comment]
Do you think it is posible to get the height of a row in the chart ? @junedchhipa how it is calculated currently ? 
Knowing so, I can adjust the size of the img... 
![Annotation 2020-06-21 140220](https://user-images.githubusercontent.com/36273997/85224061-d7e20880-b3c7-11ea-8945-75c9c99e87c7.png)

thank you in advance

**Comment by kamilcglr** [comment]
I found a solution which is not necessarily the cleanest, but it is possible to retrieve the height of a row on the chart. In my case I retrieve the apexChartGridRow information and use it to define the height of my images. 
Here is a piece of code with vuejs
`
const para = document.getElementsByClassName('apexcharts-grid-row')
        this.$nextTick(function() {
          // Code that will run only after the
          // entire view has been re-rendered
          setTimeout(() => {
            this.apexChartGridRow =
              para.length > 0
                ? Math.round(para[0].attributes.height.value * 100) / 100
                : 0
          }, 500)
        })
`

**Comment by havardl** [comment]
@kamilcglr your code seems to get the height, but how do you insert the images for each row?

**Comment by kamilcglr** [comment]
Hello @havardl,

We use the Vuetify framework.

We have two columns (v-col) in a row (v-row) :
- the first column is narrower and allows you to position the images in rows (v-row)
- the second column contains the apexChart graph

By retrieving the height of the apexChartGridRow we can define the same heights in the column containing the images.
It is a little bit tacky but we did not find any other solution.
It's been in production since August and it's working well.

**Comment by kamilcglr** [comment]
By the way we changed our code to: 
```
mounted: (chartContext, config) => {
                try {
                  const divContainer = document.getElementById(this.targetId)
                  const para = divContainer.getElementsByClassName('apexcharts-grid-row')
                  this.apexChartGridRow =
                    para.length > 0 ? Math.round(para[0].attributes.height.value * 100) / 100 : 0
                  this.hideDownloadButton()
                } catch (e) {
                  // Chart is not displayed, so error raised
                }
              },
updated: (chartContext, config) => {
  try {
    const divContainer = document.getElementById(this.targetId)
    const para = divContainer.getElementsByClassName('apexcharts-grid-row')
    this.apexChartGridRow =
      para.length > 0 ? Math.round(para[0].attributes.height.value * 100) / 100 : 0
    this.hideDownloadButton()
  } catch (e) {
    // Chart is not displayed, so error raised
  }
```
